db/helpers/common.js

```diff
@@ -13,9 +13,12 @@
  */
 'use strict'; // eslint-disable-line strict
 
-const pub = require('../../cache/redisCache').client.pub;
```

Comment:
Actually, we want to remove all the realtime stuff from this file and only want to use what ever is under the realtime folder for pub/sub.